packages/typespec-go/src/tcgcadapter/types.ts

```diff
@@ -3,13 +3,13 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import * as naming from '../../../naming.go/src/naming.js';
-import * as go from '../../../codemodel.go/src/index.js';
+import { uncapitalize } from '@azure-tools/codegen';
```

Comment:
In the future, it would be ideal to omit such refactoring from PRs with functional changes to reduce the delta, making it easier for reviewers to focus on the actual changes.

Reply:
sure. do you think i need to revert it from this pr?

Reply:
Agreed on the consistency. I will work on updating CI to enforce this.

Reply:
reverted all format change.